multisig/proposals/arbitrumsepolia/vip-001/index.ts

```diff
@@ -0,0 +1,181 @@
+const CHAINLINK_USDC_FEED = "0x0153002d20B96532C639313c2d54c3dA09109309";
+const CHAINLINK_USDT_FEED = "0x80EDee6f667eCc9f63a0a6f55578F870651f06A4";
+const CHAINLINK_ARB_FEED = "0xD1092a65338d049DB68D7Be6bD89d17a0929945e";
+
+const MOCK_WBTC = "0xFb8d93FD3Cf18386a5564bb5619cD1FdB130dF7D";
+const MOCK_WETH = "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73";
+const MOCK_USDT = "0xf3118a17863996B9F2A073c9A66Faaa664355cf8";
+const MOCK_USDC = "0x86f096B1D970990091319835faF3Ee011708eAe8";
+const MOCK_ARB = "0x4371bb358aB5cC192E481543417D2F67b8781731";
+
+const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
+export const BOUND_VALIDATOR = "0xfe6bc1545Cc14C131bacA97476D6035ffcC0b889";
+
+const vip001 = () => {
+  return makeProposal([
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "pause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "unpause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "setOracle(address,address,uint8)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "enableOracle(address,uint8,bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "setTokenConfig(TokenConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.CHAINLINK_ORACLE, "setTokenConfig(TokenConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.CHAINLINK_ORACLE, "setDirectPrice(address,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [BOUND_VALIDATOR, "setValidateConfig(ValidateConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WBTC, CHAINLINK_BTC_FEED, 144000]],
```

Comment:
The heartbeat for BTC/USD in Arbitrum sepolia is `120`. `144` seconds seems enough, wdyt?

Same comment for ETH/USD

Reply:
I think setting `144` seconds would be enough. But as i can see that for 1 minute heartbeat feeds, we have set it at 100 seconds [here](https://github.com/VenusProtocol/vips/blob/b8325d5d0a61136f12cac100b4c8540f1d1b6c93/multisig/proposals/opbnbmainnet/vip-001/index.ts#L16)
So, what if we set it at `180` seconds?

Reply:
Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3